include/lemlib/logger/telemetrySink.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "baseSink.hpp"
```

Comment:
Use full file path in include

```suggestion
#include "lemlib/logger/baseSink.hpp"
```